FrameworkInternals/optionalModules.py

```diff
@@ -172,65 +172,64 @@ def enableModule(moduleName, tag="master", serverString=""):
 	if not _checkTagExists(moduleInfo[moduleName]["url"], tag): return
 
 	baseDirectory = os.getcwd()
-	fwInternalsDir = baseDirectory + os.path.sep + "FrameworkInternals"
-	os.chdir(fwInternalsDir)
+	fwInternalsDir = os.path.join(baseDirectory, "FrameworkInternals")
+        enabledModulesDir = os.path.join(fwInternalsDir, "EnabledModules")
 
 	# Check first if module is maybe already present with different version
 	#
-	tagFileName = "EnabledModules/"+moduleName+".tag"

```

Comment:
Why renaming this variable? 
It stores the name of the tag file so the original name seems far more appropriate.